homeworks/taron_hakobyan/Homework(Streams)/Main.java

```diff
@@ -0,0 +1,50 @@
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class Main {
+
+    public static void main(String[] args) {
+        System.out.println();
+        ArrayList<Pair> pairs = new ArrayList<Pair>();
+        File file = new File("newFile1.txt");
+
+        try (
+                FileInputStream fileInputStream = new FileInputStream(file);
+                BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+        ) {
+            for (int isLine = 0; isLine < MyFunctions.getLinesCount(file); isLine++) {
+                String nextLine = bufferedReader.readLine();
+                if (!MyFunctions.rightFormat(nextLine)) {
+                    throw new InvalidFormatException("Invalid Format");
+                } else {
```

Comment:
Sorry, goingToAddInLetter and goingToAddInNumber are useless , if statement is unnecessary